packages/publisher/s3/src/PublisherS3.ts

```diff
@@ -90,11 +90,12 @@ export default class PublisherS3 extends PublisherBase<PublisherS3Config> {
   }
 
   generateCredentials(): Credentials | undefined {
-    const accessKeyId = this.config.accessKeyId || process.env.AWS_ACCESS_KEY_ID;
-    const secretAccessKey = this.config.secretAccessKey || process.env.AWS_SECRET_ACCESS_KEY;
+    const accessKeyId = this.config.accessKeyId;
+    const secretAccessKey = this.config.secretAccessKey;
```

Comment:
Thanks for the PR! Could we re-add the process.env fallbacks here, so that these are included if they're not in the config (CI could be a use case here)?

Reply:
This GitHub Action does use environment variables! https://github.com/aws-actions/configure-aws-credentials

Reply:
@VerteDinde  if the credentials are not in the config and present in the `process.env`, the SDK **will** use them without needing to pass them explicitly.

THis is the recommended way. [See doc](https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/setting-credentials-node.html)